test/functional/eval/system_spec.lua

```diff
@@ -464,7 +491,7 @@ describe('systemlist()', function()
     after_each(delete_file(fname))
 
     it('replaces NULs by newline characters', function()
-      eq({'part1\npart2\npart3'}, eval('systemlist("cat '..fname..'")'))
+      eq({'part1\npart2\npart3'}, eval([[systemlist('"cat" "]]..fname..[["')]]))
```

Comment:
Is quoting like this now _required_, or is this change just to test #7698 ?

Reply:
It's not required but #7698 had it. Other tests using cat aren't quoted and work fine.